### ISSUE TITLE
Photos on boefjes are large / load slow

### ISSUE THREAD

**Issue by sigio** [minvws/nl-kat-coordination#117]
The cat-photos on boefjes (icons) are full-size jpeg's and load slowly.... they need to be resized to the actually used size and compressed.

Example: https://kat.rdosecurity.nl/kat-alogus/plugins/dicom/cover.jpg (250kb, 1300x900px)

**Comment by underdarknl** [comment]
agreed! I think we have done this for some, but we should keep new additions small when we accept them.
Also, we should check if the cache headers are set properly.
